components/navbar/style.css

```diff
@@ -0,0 +1,81 @@
+header {
+    background-color: #036;
+    border-bottom: 2px solid #fcba19;
+    padding: 0 30px 0 30px;
+    color: #fff;
+    display: flex;
+    height: 65px;
+    top: 0px;
+    position: fixed;
+    width: 100%;
+    -webkit-box-shadow: 0 6px 8px -4px #b3b1b3;
+    -moz-box-shadow: 0 6px 8px -4px #b3b1b3;
+    box-shadow: 0 6px 8px -4px #b3b1b3;
+  }
+  
+  header h1 {
+    font-family: 'PT Sans';
+    font-weight: normal;  /* 400 */
+    margin: 5px 5px 0 18px;
+    visibility: hidden;
+  }
+  
+  header .banner {
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+    margin: 0 10px 0 0;
+    /* border-style: dotted;
+    border-width: 1px;
+    border-color: lightgrey; */
+  }
+  
+  header .other {
+    display: flex;
+    align-items: center;
+    flex-grow: 1;
+    /* border-style: dotted;
+    border-width: 1px;
+    border-color: lightgrey; */
+  }
+
+  header .nav-btn {
+    top: 15px;
+    right: 10px;
+    z-index: 100;
```

Comment:
I may be wrong here, but i think `top, right, z-index` have no effect here since this is not a `positioned` element. 

top/right only works on positioned elements `position: relative, position: absolute, position: fixed` etc.

Reply:
Fixed.